core/CMake_symbol_check.cmake

```diff
@@ -47,8 +66,22 @@ execute_process(COMMAND
   ERROR_VARIABLE readelf_error
   OUTPUT_VARIABLE output
   )
+
+is_i3450_binutils_version(is_i3450)
```

Comment:
This should be below the comment which talks about it

Reply:
Done.